tests/acceptance/features/keystone.feature

```diff
@@ -17,6 +17,11 @@ Feature: Get the list of expired users from the IdM.
       When I request a list of trial users from the Keystone
       Then the Keystone returns a list with all the trial users registered
 
+    Scenario: 03: Get the list of community users
+      Given a valid token from the Keystone
```

Comment:
I think keystone.feature will be removed since the functionality for get users and expired users are in the others features